permissions/component-jenkins-maven-plugin.yml

```
@@ -1,7 +1,7 @@
 ---
 name: "jenkins-maven-plugin"
 paths:
-- "com/nirima/jenkins/repository/jenkins-maven-plugin"
+- "com/nirima/jenkins/repository"
```

I think these are the paths you should have, the group ID for the plugin should be changed I think for consistency, (to make it work right now you would replace `com/nirima/jenkins/repository/repository` with `jenkins/repository` in the suggestion below:
 
```suggestion
- "com/nirima/jenkins/repository/pom"
- "com/nirima/jenkins/repository/jenkins-maven-plugin"
- "com/nirima/jenkins/repository"
- "com/nirima/jenkins/repository/repository"
```